client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -841,6 +847,7 @@ export class CheckoutThankYou extends Component<
 					displayMode={ displayMode }
 					purchases={ purchases }
 					isRedesignV2={ isRedesignV2( this.props ) }
+					currency={ receipt.data?.currency }
```

Comment:
Can we use the currency from each individual purchase instead?

If not, we should pass down the whole receipt instead of just the list of purchases.

Reply:
Feel free to disregard, I just saw the note in D116751-code.